src/client/components/Reviews.js

```diff
@@ -1,5 +1,56 @@
-import React from "react";
+import React, { useState, useContext } from "react";
+import Header from "./Header";
+import Footer from "./Footer";
+import DataContext from "./DataContext";
+import { Link } from "react-router-dom";
 
 export default function Reviews() {
-  return <h2>Reviews</h2>;
+  const { meals, reviews, loading, error } = useContext(DataContext);
+  if (loading) return <h3>Loading...</h3>;
+  if (error) return <p>{error}</p>;
+
+  function reviewStars(stars){
```

Comment:
When you start extracting code into their own modules, it becomes easier to remember what code you already have modules for so you don't have to repeat yourself! Less code = less bugs.